specificatie/openapi.yaml

```diff
@@ -216,9 +452,16 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/Nationaliteithistorie'
+    NationaliteithistorieCollectie:
```

Comment:
NationaliteithistorieCollectie en NationaliteithistorieHalCollectieEmbedded zijn precies hetzelfde. Eentje verwijderen?

Reply:
De ene (embedded) heet zo vanuit consistente naamgeving vanwege Hal. De andere wordt gebruikt in de Post (plain Json) Als er straks na de pilot een keuze gemaakt is valt sowieso 1 van de twee af. Ik zou het voor de pilot zo laten.